File: SDKs/amazonaws.com/lightsail/2016-11-28/python/src/sdk/models/shared/getrelationaldatabaseblueprintsresult.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from . import relationaldatabaseblueprint


@dataclass_json
@dataclass
class GetRelationalDatabaseBlueprintsResult:
    blueprints: Optional[List[relationaldatabaseblueprint.RelationalDatabaseBlueprint]] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'blueprints' }})
    next_page_token: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'nextPageToken' }})
    

File: SDKs/amazonaws.com/lightsail/2016-11-28/typescript/src/sdk/models/shared/getrelationaldatabaseblueprintsresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { RelationalDatabaseBlueprint } from "./relationaldatabaseblueprint";



export class GetRelationalDatabaseBlueprintsResult extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=blueprints", elemType: RelationalDatabaseBlueprint })
  blueprints?: RelationalDatabaseBlueprint[];

  @SpeakeasyMetadata({ data: "json, name=nextPageToken" })
  nextPageToken?: string;
}
